### PR TITLE
chore(virtualRepeat): Address review comments on #5561

### PR DESCRIPTION
**Pull request by kseamon**

### CODE REVIEW

**Comment by kseamon** [comment]
@jelbourn @ThomasBurleson

**Comment by hillerstorm** [comment]
Since the original issue (#5561) totally broke autocomplete (see #5606), either add some tests to address that problem or revert it completely... please don't just blindly merge this without actually testing it first :innocent:

**Comment by kseamon** [comment]
Not actually sure how the other one was merged. If I did it then it was not intentional.

**Comment by hillerstorm** [comment]
@kseamon nah, I think it was @ThomasBurleson who merged it

File: src/components/virtualRepeat/virtual-repeater.spec.js [hunk 1]
```diff
@@ -522,11 +524,13 @@ describe('<md-virtual-repeat>', function() {
   });
 
   it('should recheck container size on window resize', function() {
+    console.log($mdUtil.debounce);
```
Comment (jelbourn) [inline src/components/virtualRepeat/virtual-repeater.spec.js]: console.log

File: src/components/virtualRepeat/virtual-repeater.spec.js [hunk 2]
```diff
@@ -522,11 +524,13 @@ describe('<md-virtual-repeat>', function() {
...
+    spyOn($mdUtil, 'debounce').and.callFake(function(i) {return i;});
```
Comment (jelbourn) [inline src/components/virtualRepeat/virtual-repeater.spec.js]: `function(i) {return i;}` > `angular.identity` ?

**Comment by jelbourn** [comment]
@kseamon general LGTM on the code, though I'm not sure how the other PR was merged. 

What's the story with autocomplete?

**Comment by kseamon** [comment]
As I commented on the Autocomplete issue, I was not able to reproduce it on the demo page. My only guess is that because #5561 removed the explicit ng-hide check in favor of the not yet checked in $md-resize broadcasting from ng-hide and ng-show, uses cases that depended on that are broken.

File: src/components/virtualRepeat/virtual-repeater.spec.js (hunk 1 above)
Comment (kseamon) [inline src/components/virtualRepeat/virtual-repeater.spec.js]: Done.

File: src/components/virtualRepeat/virtual-repeater.spec.js (hunk 2 above)
Comment (kseamon) [inline src/components/virtualRepeat/virtual-repeater.spec.js]: Done

**Comment by kseamon** [comment]
As I noted in the issue, #5579 fixes the autocomplete issue.

**Comment by jelbourn** [comment]
@ThomasBurleson LGTM to merge

**Pull request closed without merging**